package.json

```diff
@@ -948,6 +949,9 @@
     "@npkg/tinymce-plugins": {
       "version": "*"
     },
+    "@nutui/nutui": {
+      "version": "4.3.13"
```

Comment:
should change to `*` for all future versions.

Reply:
You can just delete this configuration. The `@nutui` scope has already been added above, which will include this package in the whitelist.